Speekins/overlook
# Write HTML for customer view

#### What's this PR do?
This merge will provide the general layout of HTML and some CSS for the `customer` view
#### Where should the reviewer start?
Pull down branch, run `npm start` in both `overlook` and `overlook-api` root directories. Navigate to the browser using the local url provided by the `overlook` server.
#### How should this be manually tested?
Navigate to the application in the browser (url provided by `overlook` local api). Check elements by using dev tools
#### Screenshots
![image](https://user-images.githubusercontent.com/74210902/201488307-988dd436-c639-4d72-b63d-87b0121b203d.png)
